### PR TITLE
feat: upgrade argo-admin 0.6.0

### PR DESCRIPTION
**Pull request by alanthai**
https://github.com/Shopify/app-extension-libs/issues/917

Related: https://github.com/Shopify/argo-admin-cli/pull/28

## 🎩 instructions

1. [Tophat instructions](https://github.com/Shopify/app-extension-libs/wiki/Test-argo-admin-cli-changes#-tophat-instructions) and additionally change `argo-admin` to alpha as well.

    ```diff
    "devDependencies": {
    -    "@shopify/argo-admin-cli": "latest",
    +    "@shopify/argo-admin-cli": "git+https://github.com/Shopify/argo-admin-cli.git#feat/upgrade-argo-admin-0.6.0"
    },

    "dependencies": {
    -    "@shopify/argo-admin": "latest",
    -    "@shopify/argo-admin-host": "latest",
    +    "@shopify/argo-admin": "0.6.0-alpha.0",
    +    "@shopify/argo-admin-host": "0.6.0-alpha.0",
    },
    ```

1. Check generate script now returns type `PRODUCT_SUBSCRIPTION`
1. Do quick smoke test

### CODE REVIEW

**Review by iainmcampbell — COMMENTED** [review]
getting

```
npm ERR! error: pathspec 'chore/upgrade-argo-admin-0.6.0-test' did not match any file(s) known to git
```

when i try to install

```
    "@shopify/argo-admin-cli": "git+https://github.com/Shopify/argo-admin-cli.git#chore/upgrade-argo-admin-0.6.0-test",
```

is there something i need to do to get https github urls to work?

File: scripts/generate/constants.ts [hunk 1]
```diff
@@ -1,3 +1,6 @@
 // corresponds to Extension::Model::Type.identifier
-// https://github.com/Shopify/shopify-app-cli-extensions/tree/master/lib/project_types/extension/models/types
-export const extensionTypes = ['SUBSCRIPTION_MANAGEMENT'];
+// https://github.com/Shopify/shopify-app-cli/tree/master/lib/project_types/extension/models/types
+export const extensionTypes = [
+  'PRODUCT_SUBSCRIPTION',
+  'CHECKOUT_POST_PURCHASE',
```
Comment (henrytao-me) [inline scripts/generate/constants.ts:5]: Why do we need `CHECKOUT_POST_PURCHASE` here? It should belong to argo-checkout-template.

**Comment by henrytao-me** [comment]
@iainmcampbell it's better to checkout argo-admin-cli local, build it, then use local dependencies instead of git url

```
"@shopify/argo-admin-cli": "file:../argo-admin-cli",
```

**Review by henrytao-me — COMMENTED** [review]
🎩-ed locally with https://github.com/Shopify/argo-admin-cli/pull/28. LGTM. 

I have one comment to clarify.

**Review by iainmcampbell — APPROVED** [review]

**Review by henrytao-me — APPROVED** [review]
LGTM :shipit:

**Pull request merged**